pkg/policy/policiesSchema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "type": "string",
+      "enum": [
+        "v1"
+      ]
+    },
+    "message": {
+      "type": "string",
+      "enum": [
+        "Should have exact one policy set as default",
+        "All policy names should be unique",
+        "All rule identifiers should be unique"
+      ]
+    },
+    "policies": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": [
+          "name"
+        ],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "isDefault": {
+            "type": "boolean"
+          },
+          "message": {
+            "type": "string"
+          },
+          "rules": {
+            "items": {
+              "type": "object",
+              "properties": {
+                "identifier": {
```

Comment:
where is the check that validates rule identifier exists in either custom rules or default rules?

Reply:
I think this is one of the abilities that we are losing due to JSON Schema limitations.

https://github.com/json-schema-org/json-schema-spec/wiki/Scope-of-JSON-Schema-Validation